spec/src/main/asciidoc/microprofile-openapi-spec.asciidoc

```diff
@@ -72,8 +72,8 @@ which means that vendors implementing the MP OpenAPI specification must also imp
 the MP Config specification.
 
 There are various ways to inject these configuration values into an MP OpenAPI
-framework, including the https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/configsources.asciidoc#default-configources[default ConfigSource] as well as
-https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/configsources.asciidoc#custom-configsources[custom ConfigSource].
+framework, including the https://github.com/eclipse/microprofile-config/blob/main/spec/src/main/asciidoc/configsources.asciidoc#default-configources[default ConfigSource] as well as
+https://github.com/eclipse/microprofile-config/blob/main/spec/src/main/asciidoc/configsources.asciidoc#custom-configsources[custom ConfigSource].
```

Comment:
Link to the MP config spec, not the source code: https://download.eclipse.org/microprofile/microprofile-config-3.1/microprofile-config-spec-3.1.html